docs/pages/custom-builds/schema.mdx

```diff
@@ -570,6 +570,97 @@ EAS provides a set of built-in reusable functions that you can use in a workflow
+   - if your build uses credentials (for both `internal` and `store` [distribution](/eas/json/#distribution) builds):
+     - [`eas/checkout`](#eascheckout)
+     - [`eas/install_node_modules`](#easinstall_node_modules)
+     - [`eas/get_credentials_for_build_triggered_by_github_integration`](#easget_credentials_for_build_triggered_by_github_integration)
+     - [`eas/prebuild`](#easprebuild)
+     - [`eas/configure_eas_update`](#easconfigure_eas_update)
+     - [`eas/inject_android_credentials`](#easinject_android_credentials)
+     - [`eas/configure_android_version`](#easconfigure_android_version)
+     - [`eas/run_gradle`](#easrun_gradle)
+     - [`eas/find_and_upload_build_artifacts`](#easfind_and_upload_build_artifacts)
+
+You can replace the `eas/build` command call by using these steps in your YAML configuration file:
+
+<BoxLink
+  title="ios-simulator-build.yml"
+  description="View the steps executed behind the scenes by the `eas/build` function for an iOS simulator build in our example repository."
+  Icon={GithubIcon}
+  href="https://github.com/expo/eas-custom-builds-example/blob/main/.eas/build/ios-simulator-build.yml"
+/>
+<BoxLink
+  title="ios-credentials-build.yml"
+  description="View the steps executed behind the scenes by the `eas/build` function for an iOS build with credentials in our example repository."
+  Icon={GithubIcon}
+  href="https://github.com/expo/eas-custom-builds-example/blob/main/.eas/build/ios-build-with-credentials.yml"
+/>
+<BoxLink
+  title="android-build-without-credentials.yml"
+  description="View the steps executed behind the scenes by the `eas/build` function for an Android build without credentials in our example repository."
+  Icon={GithubIcon}
+  href="https://github.com/expo/eas-custom-builds-example/blob/main/.eas/build/android-build-without-credentials.yml"
+/>
+<BoxLink
+  title="android-build-with-credentials.yml"
+  description="View the steps executed behind the scenes by the `eas/build` function for an Android build with credentials in our example repository."
+  Icon={GithubIcon}
+  href="https://github.com/expo/eas-custom-builds-example/blob/main/.eas/build/android-build-with-credentials.yml"
+/>
+
+Known limitations:
+
+- This function won't configure updates for you properly if you still use the legacy classic updates. You should use the EAS Update instead. Check our [migration guide](/eas-update/migrate-from-classic-updates/) for more information.
+- Currently it doesn't accept any inputs and the resolved build process will be configured based on your build profile from [**eas.json**](/eas/json/) settings.
+- The build process produced by `eas/build` is not configurable and you can't customize it. If you need to customize the build process you should use the subset of functions and steps that are executed behind the scenes by this function and configure them manually in your YAML configuration file as shown in the examples above.
```

Comment:
```suggestion
- The build process produced by `eas/build` is not configurable and you can't customize it. If you need to customize the build process, use the subset of functions and steps that are executed behind the scenes by this function and configure them manually in the YAML configuration file, as shown in the examples above.
```

- Which examples is the last line referring to?

Reply:
```md
##### Android

When a build configuration is using [`withoutCredentials`](/eas/json/#withoutcredentials):

- [`eas/checkout`](#eascheckout)
- [`eas/install_node_modules`](#easinstall_node_modules)
- [`eas/prebuild`](#easprebuild)
- [`eas/configure_eas_update`](#easconfigure_eas_update)
- [`eas/run_gradle`](#easrun_gradle)
- [`eas/find_and_upload_build_artifacts`](#easfind_and_upload_build_artifacts)

When a build configuration uses credentials (for both `internal` and `store` [distribution](/eas/json/#distribution) builds):

- [`eas/checkout`](#eascheckout)
- [`eas/install_node_modules`](#easinstall_node_modules)
- [`eas/get_credentials_for_build_triggered_by_github_integration`](#easget_credentials_for_build_triggered_by_github_integration)
- [`eas/prebuild`](#easprebuild)
- [`eas/configure_eas_update`](#easconfigure_eas_update)
- [`eas/inject_android_credentials`](#easinject_android_credentials)
- [`eas/configure_android_version`](#easconfigure_android_version)
- [`eas/run_gradle`](#easrun_gradle)
- [`eas/find_and_upload_build_artifacts`](#easfind_and_upload_build_artifacts)


##### iOS

When a build configuration is using [`withoutCredentials`](/eas/json/#withoutcredentials) or [`simulator`](/eas/json/#simulator):
- [`eas/checkout`](#eascheckout)
- [`eas/install_node_modules`](#easinstall_node_modules)
- [`eas/prebuild`](#easprebuild)
- Install pods using the `pod install` command
- [`eas/configure_eas_update`](#easconfigure_eas_update)
- [`eas/generate_gymfile_from_template`](#easgenerate_gymfile_from_template)
- [`eas/run_fastlane`](#easrun_fastlane)
- [`eas/find_and_upload_build_artifacts`](#easfind_and_upload_build_artifacts)


When a build configuration uses credentials (for both `internal` and `store` [distribution](/eas/json/#distribution) builds):

- [`eas/checkout`](#eascheckout)
- [`eas/install_node_modules`](#easinstall_node_modules)
- [`eas/get_credentials_for_build_triggered_by_github_integration`](#easget_credentials_for_build_triggered_by_github_integration)
- [`eas/resolve_apple_team_id_from_credentials`](#easresolve_apple_team_id_from_credentials)
- [`eas/prebuild`](#easprebuild)
- Install pods using the `pod install` command
- [`eas/configure_eas_update`](#easconfigure_eas_update)
- [`eas/configure_ios_credentials`](#easconfigure_ios_credentials)
- [`eas/configure_ios_version`](#easconfigure_ios_version)
- [`eas/generate_gymfile_from_template`](#easgenerate_gymfile_from_template)
- [`eas/run_fastlane`](#easrun_fastlane)
- [`eas/find_and_upload_build_artifacts`](#easfind_and_upload_build_artifacts)
```

I meant these examples

Reply:
Thanks for the clarification!